src/components/pages/CardView.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="container">
+    <card :cdata="deepUrlCard"/>
+  </div>
+</template>
+
+<script>
+import { mapState } from "vuex";
+import Card from "../../components/widgets/Card";
+import router from "../../router";
+import * as actions from "../../store/actions";
+
+export default {
+  name: "creator",
+  components: { Card },
+  data() {
+    return {
+      dispatchCardCount: 0,
+      dispatchCardDeep: 0
+    };
+  },
+  computed: {
+    ...mapState(["deepUrlCard"])
+  },
+  mounted() {
+    console.log("loading Started");
+    let currentPath = this.$route.path;
+    let tokenId = currentPath.substring(
+      currentPath.lastIndexOf(":") + 1,
+      currentPath.length
```

Comment:
Im not sure this is needed? In vue you are call `this.$route.params.id` if you change you route path to be `/card/:id`? Maybe I am missing what you are trying to achieve here.

Reply:
I didnt know about `this.$route.params.id` much better solution to what I was doing! updated the component and the router to achieve your implementation.